x-pack/plugins/upgrade_assistant/server/lib/reindexing/reindex_actions.ts

```diff
@@ -248,8 +251,7 @@ export const reindexActionsFactory = (
 
     async getFlatSettings(indexName: string) {
       const flatSettings = (await callCluster('transport.request', {
-        // TODO: set `&include_type_name=true` to false in 7.0
-        path: `/${encodeURIComponent(indexName)}?flat_settings=true`,
```

Comment:
Great, I was going to fix this one up today. Though a 7.0 node cannot join a cluster with pre-6.0 indices so this edge case shouldn't ever bit hit anyways. Still good to use this flag in case I'm wrong though.